ethcore/wasm/src/tests.rs

```diff
@@ -415,36 +415,121 @@ fn storage_read() {
 	assert_eq!(Address::from(&result[12..32]), address);
 }
 
+macro_rules! reqrep_test {
```

Comment:
This could be just a function that accepts `code` and `input`

Reply:
but it uses macro, the testfile content should be known at compile time, no runtime loading there

Reply:
Right, that's exactly why I was suggesting making testfile content (`code`) an argument to the function.